app/graphql/mutations/digital_object/update_projects.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+module Mutations
+  module DigitalObject
+    class UpdateProjects < Mutations::BaseMutation
+      argument :id, ID, required: true
+      argument :primary_project, Inputs::StringKey, required: true
```

Comment:
```suggestion
      argument :primary_project, Inputs::StringKey, required: false
```
Should this be `required: false` so users can use this mutation to update only the primary_project without having to supply other_projects?